.ci/Jenkinsfile

```diff
@@ -77,9 +77,11 @@ pipeline {
           stage('Test') {
             steps {
               withGithubNotify(context: "Test-${PLATFORM}") {
-                dir("${BASE_DIR}"){
-                  withGoEnv(){
-                    goTestJUnit(options: '-v ./...', output: 'junit-report.xml')
+                withMageEnv(){
+                  dir("${BASE_DIR}"){
+                    withEnv() {
+                      cmd(label: 'Go unitTest', script: 'mage unittest')
```

Comment:
Is `unittest` the formal name or camelcase based (`unitTest`)?

Reply:
fixed